lib/rindle.js

```diff
@@ -104,18 +103,19 @@ exports.wait = function(stream, callback) {
  *   console.log('The file contains: ' + data);
  * });
  */
-exports.extract = function(stream, callback) {
-  'use strict';
-
-  var chunks = '';
-
-  stream.on('data', function(chunk) {
-    chunks += chunk;
-  });
-
-  return exports.wait(stream).then(function() {
-    return chunks;
-  }).nodeify(callback);
+exports.extract = function (stream, callback) {
+	var chunks = '';
+
+	stream.on('data', function (chunk) {
+		chunks += chunk;
+	});
+
+	return exports
+		.wait(stream)
+		.then(function () {
+			return chunks;
+		})
```

Comment:
Worth changing as follows, but feel free to ignore given it wasn't like that before and it's reasonable to expect we'll switch to native promises at some point:

```suggestion
		.return(chunks)
```

Reply:
Yeah my assumption would be we'll go to native promises and async/await at some point so I'd avoid adding bluebird sugar - utility functions are fair game imo though